node/index.ts

```diff
@@ -1003,3 +1195,168 @@ export async function GroupCipher_Decrypt(
 ): Promise<Buffer> {
   return SC.GroupCipher_Decrypt(name._unsafeGetNativeHandle(), store, message);
 }
+
+export class SealedSenderDecryptionResult {
+  private readonly nativeHandle: SignalClient.SealedSenderDecryptionResult;
+
+  private constructor(nativeHandle: SignalClient.SealedSenderDecryptionResult) {
+    this.nativeHandle = nativeHandle;
+  }
+
+  static _fromNativeHandle(
+    nativeHandle: SignalClient.SealedSenderDecryptionResult
+  ): SealedSenderDecryptionResult {
+    return new SealedSenderDecryptionResult(nativeHandle);
+  }
+
+  message(): Buffer {
+    return SC.SealedSenderDecryptionResult_Message(this.nativeHandle);
+  }
+
+  senderE164(): string | null {
+    return SC.SealedSenderDecryptionResult_GetSenderE164(this.nativeHandle);
+  }
+
+  senderUuid(): string {
+    return SC.SealedSenderDecryptionResult_GetSenderUuid(this.nativeHandle);
+  }
+
+  deviceId(): number {
+    return SC.SealedSenderDecryptionResult_GetDeviceId(this.nativeHandle);
+  }
+}
+
+export class CiphertextMessage {
+  private readonly nativeHandle: SignalClient.CiphertextMessage;
+
+  private constructor(nativeHandle: SignalClient.CiphertextMessage) {
+    this.nativeHandle = nativeHandle;
+  }
+
+  static _fromNativeHandle(
+    nativeHandle: SignalClient.CiphertextMessage
+  ): CiphertextMessage {
+    return new CiphertextMessage(nativeHandle);
+  }
+
+  serialize(): Buffer {
+    return SC.CiphertextMessage_Serialize(this.nativeHandle);
+  }
+
+  type(): number {
+    return SC.CiphertextMessage_Type(this.nativeHandle);
+  }
+}
+
+export function ProcessPreKeyBundle(
```

Comment:
These top-level functions should be lowerCamelCase, yeah? We can use the Swift names for them.